File: Nx3CS/bin/test/src/nx3/ETie.cs

#pragma warning disable 109, 114, 219, 429, 168, 162
namespace nx3{
	public  class ETie : global::haxe.lang.Enum {
		static ETie() {
			#line 8 "F:\\nx3\\src\\nx3\\ETie.hx"
			global::nx3.ETie.constructs = new global::Array<object>(new object[]{"Tie", "Gliss"});
		}
		public    ETie(global::haxe.lang.EmptyObject empty) : base(global::haxe.lang.EmptyObject.EMPTY){
			unchecked {
			}
			#line default
		}
		
		
		public    ETie(int index, global::Array<object> @params) : base(index, @params){
			unchecked {
			}
			#line default
		}
		
		
		public static  global::Array<object> constructs;
		
		public static   global::nx3.ETie Tie(global::nx3.EDirectionUAD direction, int level){
			unchecked {
				#line 10 "F:\\nx3\\src\\nx3\\ETie.hx"
				return new global::nx3.ETie(((int) (0) ), ((global::Array<object>) (new global::Array<object>(new object[]{direction, level})) ));
			}
			#line default
		}
		
		
		public static   global::nx3.ETie Gliss(global::nx3.EDirectionUAD direction, int levelLeft, int levelRight){
			unchecked {
				#line 11 "F:\\nx3\\src\\nx3\\ETie.hx"
				return new global::nx3.ETie(((int) (1) ), ((global::Array<object>) (new global::Array<object>(new object[]{direction, levelLeft, levelRight})) ));
			}
			#line default
		}
		
		
		public static  new object __hx_createEmpty(){
			unchecked {
				#line 8 "F:\\nx3\\src\\nx3\\ETie.hx"
				return new global::nx3.ETie(((global::haxe.lang.EmptyObject) (global::haxe.lang.EmptyObject.EMPTY) ));
			}
			#line default
		}
		
		
		public static  new object __hx_create(global::Array arr){
			unchecked {
				#line 8 "F:\\nx3\\src\\nx3\\ETie.hx"
				return new global::nx3.ETie(((int) (global::haxe.lang.Runtime.toInt(arr[0])) ), ((global::Array<object>) (global::Array<object>.__hx_cast<object>(((global::Array) (arr[1]) ))) ));
			}
			#line default
		}
		
		
	}
}

File: Nx3Java/java/src/nx3/ETie.java
package nx3;
import haxe.root.*;

@SuppressWarnings(value={"rawtypes", "unchecked"})
public  class ETie extends haxe.lang.Enum
{
	static 
	{
		nx3.ETie.constructs = new haxe.root.Array<java.lang.String>(new java.lang.String[]{"Tie", "Gliss"});
	}
	public    ETie(int index, haxe.root.Array<java.lang.Object> params)
	{
		super(index, params);
	}
	
	
	public static  haxe.root.Array<java.lang.String> constructs;
	
	public static   nx3.ETie Tie(nx3.EDirectionUAD direction, int level)
	{
		return new nx3.ETie(((int) (0) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{direction, level})) ));
	}
	
	
	public static   nx3.ETie Gliss(nx3.EDirectionUAD direction, int levelLeft, int levelRight)
	{
		return new nx3.ETie(((int) (1) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{direction, levelLeft, levelRight})) ));
	}
	
	
}


